src/Raven.Server/Documents/Replication/ReplicationItems/AttachmentReplicationItem.cs

```diff
@@ -121,13 +122,20 @@ protected override ReplicationBatchItem CloneInternal(JsonOperationContext conte
 
             var item = new AttachmentReplicationItem
             {
-                Base64Hash = Base64Hash,
-                ContentType = ContentType,
-                Name = Name,
-                Key = Key,
+                ContentType = ContentType.Clone(context),
+                Name = Name.Clone(context),
                 Stream = stream
             };
 
+            var baseMem = Base64Hash.CloneToJsonContext(context, out item.Base64Hash);
+            var keyMem = Key.CloneToJsonContext(context, out item.Key);
+
+            ToDispose(new DisposableAction(() =>
```

Comment:
Shouldn't this be attached to item.Dispose?

Reply:
I think it is..